.github/workflows/main.yml

```diff
@@ -32,8 +32,10 @@ jobs:
           # --exclude .*\.linkedin\.com/.* -> Excluded because Linkedin just responds with 429 Too Many Requests.
           # --exclude .*\.faro\.com/.* -> Excluded because link check fails on GH Action but the reason is not known.
           # --exclude .*\.humanprofiler\.com/.* -> lychee states "Network error: No status code" but URLs are OK even if slow
+          # --exclude .*\.m-profiler\.com/.* -> lychee states "Network error: No status code" but URLs are OK even if slow
+          # --exclude .*\.web.truphone.com/careers.* -> lychee states "Network error: No status code" for careers website but URL is OK
```

Comment:
Since it was just the careers page giving this error, I felt that also excluding the main website would further cloud our awareness of whether it was working or not.